dataland-frontend/src/components/pages/DataPrivacy.vue

```diff
@@ -1,38 +1,1534 @@
+          die Verarbeitung Ihrer personenbezogenen Daten durch uns und über Ihre Rechte aus dem Datenschutzrecht geben.
+          Welche Daten im Einzelnen verarbeitet werden, richtet sich maßgeblich nach Ihrem konkreten Nutzungsverhalten.
+          Daher werden unter Umständen nicht alle Teile dieser Informationen auf Sie zutreffen.</span
+        >
+      </p>
+
+      <h1
+        style="
+          margin-top: 0cm;
+          margin-right: 0cm;
+          margin-bottom: 12pt;
+          margin-left: 1cm;
+          text-align: justify;
+          text-indent: -1cm;
+          line-height: 133%;
+        "
+      >
+        1.<span style="font: 7pt &quot;Times New Roman&quot;"
+          >&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span
+        >VERANTWORTLICHE STELLE UND KONTAKTDATEN DES DATENSCHUTZBEAUFTRAGTEN
+      </h1>
+
+      <p class="MsoNormal" style="margin-bottom: 12pt; line-height: 133%">
+        <span style="font-size: 10pt; line-height: 133%"
+          >Verantwortlich für die Datenverarbeitung auf diesem Online-Angebot ist die</span
+        >
+      </p>
+
+      <p
+        class="MsoNormal"
+        style="margin-top: 0cm; margin-right: 0cm; margin-bottom: 12pt; margin-left: 35.4pt; line-height: 133%"
+      >
+        <span style="font-size: 10pt; line-height: 133%"
+          >Dataland GmbH<br />
+          Paul-Ehrlich-Straße 51<br />
+          60596 Frankfurt am Main</span
+        >
+      </p>
+
+      <p
+        class="MsoNormal"
+        style="margin-top: 0cm; margin-right: 0cm; margin-bottom: 12pt; margin-left: 35.4pt; line-height: 133%"
+      >
+        <span style="font-size: 10pt; line-height: 133%"
+          >(nachfolgend auch <b><i>„Unternehmen“ oder „wir“</i></b> genannt)</span
+        >
+      </p>
+
+      <p class="MsoNormal" style="margin-bottom: 12pt; line-height: 133%">
+        <span style="font-size: 10pt; line-height: 133%">Sie erreichen unseren Datenschutzbeauftragten unter</span>
+      </p>
+
+      <p
+        class="MsoNormal"
+        style="
+          margin-top: 0cm;
+          margin-right: 0cm;
+          margin-bottom: 12pt;
+          margin-left: 35.4pt;
+          line-height: 133%;
+          border: none;
+        "
+      >
+        <span style="font-size: 10pt; line-height: 133%"
+          >Wodianka privacy legal GmbH<br />
+          Dockenhudener Straße 12a<br />
+          22587 Hamburg<br />
+          contact@privacy-legal.de</span
+        >
+      </p>
+
+      <h1
+        style="
+          margin-top: 0cm;
+          margin-right: 0cm;
+          margin-bottom: 12pt;
+          margin-left: 1cm;
+          text-align: justify;
+          text-indent: -1cm;
+          line-height: 133%;
+        "
+      >
+        2.<span style="font: 7pt &quot;Times New Roman&quot;"
+          >&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span
+        >Verarbeitung personenbezogener Daten im Zusammenhang mit Ihrer Nutzung unserer Webseite
```

Comment:
Formatting especially ugly under mobile (huge space after the 2.)

Reply:
Answer as above.